CoverMyDotNet/Requests/PostIndicator.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using RestSharp;
+
+namespace CoverMyDotNet.Requests
+{
+	public class PostIndicator : RestRequest
+	{
+		public PostIndicator(string apiId, IndicatorAttributes indicator) : base()
+		{
+			this.JsonSerializer = new CoverMyDotNet.JsonSerializer ();
+			this.Method = Method.POST;
+			Resource = "indicators";
```

Comment:
I see in these RestRequest sub-class constructors an inconsistency where sometimes `this.` is used and sometimes not.  Is there a technical reason for this?